raw_data-to-state_district_wise_data.js

```diff
@@ -5,7 +5,9 @@ const rawData = require('./raw_data');
 console.log('Starting district wise data processing');
 try {
   const StateDistrictWiseData = rawData.raw_data.reduce((acc, row) => {
-    const isToday = moment().utcOffset(330).isSame(moment(row.dateannounced, "DD-MM-YYYY"), "day");
+    const todaysdate = moment().utcOffset(330); //set moment instance to IST timezone and fetch currect IST time
```

Comment:
Can you use "Asia/Kolkata" to get todaysdate?
So that if we are running this script in a non-UTC machine, it should work.

Reply:
The utcoffset function does exactly that thing. Moment by default is utc

Reply:
oh.. weird! In a different pc with different timezone I tried, I got different result.. 🤔

Reply:
Should I try testing again?

Reply:
it's okay. let me merge this and see.. we'll revert to the old code if this is wrong tomorrow 🤞

Reply:
Cool will check again though if there is any reason it might fall. Will let you know on this comment thread

Reply:
Thanks :)